### ISSUE TITLE
Related ayah

### ISSUE THREAD

**Issue by naveed-ahmad** [quran/quran.com-api#180]

**Comment by naveed-ahmad** [comment]
[kathir.sql.txt](https://github.com/quran/quran.com-api/files/1814034/kathir.sql.txt)

**Comment by mmahalwy** [comment]
Looked at the file. I don't understand it

**Comment by naveed-ahmad** [comment]
@mmahalwy yes column names are confusing. Here is human readable column description :) 

[ 'id', 
'source chapter no.', 
'source verse no.',
 'related chapter no.', 
'related verse no.', 
'no. of common keywords between the two verses', 
'degree of relevance (0 is lowest, 2 is strongest) '
]

**Comment by naveed-ahmad** [comment]
Took this from http://textminingthequran.com/ and we've the permission to use this dataset.

**Comment by sydfzlbshr** [comment]
@naveed-ahmad

**Comment by naveed-ahmad** [comment]
@sydfzlbshr waAlakim o salam. Not yet bro, can you parse the file and preapre a json or sqlite db? that'll be very helpful. Ill add import this to our database and expose this via api.

Next step will be show this data to users which we can discuss in new issue :)

**Comment by sydfzlbshr** [comment]
@naveed-ahmad assalamualaikum

Sorry for such a late response. I'm taking this up today.

**Comment by sydfzlbshr** [comment]
[related_verses.json.txt](https://github.com/quran/quran.com-api/files/4439396/related_verses.json.txt)
[sql_to_json.rb.txt](https://github.com/quran/quran.com-api/files/4439401/sql_to_json.rb.txt)
[related_verses.sql.txt](https://github.com/quran/quran.com-api/files/4439402/related_verses.sql.txt)

PFA SQL, Ruby Parser, and JSON
